Artemis/Artemis/Managers/LoopManager.cs

```diff
@@ -67,6 +69,9 @@ public void Start()
                 _effectManager.ChangeEffect(lastEffect);
             }
 
+            // I assume that it's safe to use ActiveKeyboard and ActifeEffect here since both is checked above
+            _keyboardBitmap = _deviceManager.ActiveKeyboard.KeyboardBitmap(_effectManager.ActiveEffect.KeyboardScale);
```

Comment:
This approach runs into some trouble when changing keyboards because the LoopManager doesn't actually restart.
Could probably clean up the DeviceManager a bit by stopping the LoopManager when changing keyboards and restarting again, which would also fix this.